keyberon/src/layout.rs

```diff
@@ -865,6 +865,16 @@ pub struct SequenceState<'a, T: 'a> {
 
 type ReleasedOneShotKeys = Vec<KCoord, ONE_SHOT_MAX_ACTIVE>;
 
+// Using a u16 for indices instead of usize.
+// Need to check against this value in code that creates layers.
+pub const MAX_LAYERS: usize = 60000;
+
+// Use heapless Vec for perf - avoid pointer indirections.
+// Use u16 for more efficient cache. 16*u16 = 4*u64 = 32 bytes.
+// Cache line is typically 64 bytes, so this takes half a cache line.
+pub const MAX_ACTIVE_LAYERS: usize = 16;
+type LayerStack = Vec<u16, MAX_ACTIVE_LAYERS>;
```

Comment:
`heapless::Vec` also stores `usize` length, so the total size of `LayerStack` seems to be 40. Are you aware of this?
![image](https://github.com/jtroo/kanata/assets/44318430/df3eec0d-370a-4345-9eb3-c0cda3fc62ae)

Reply:
Ah right! I suppose it could be reduced to 12 layers if we wanted to make the comment accurate. Or increase it to 28 to fill up a whole cache line...

12 seems plenty so I'll do that for now.

Reply:
I see you gave thumbs up reaction, so should the length be changed to 12, so it gives total 32 bytes?